openmetadata-airflow-apis/openmetadata_managed_apis/api/routes/ip.py

```diff
@@ -29,6 +32,26 @@
 
 logger = routes_logger()
 
+IPIFY_URL = "https://api.ipify.org"
```

Comment:
do we want to use IP service? I was hoping the airflow machine's ip by running local ip addresses and in some cases we can ask admin to configure what IP they want to show

Reply:
tried to test different approaches picking them up from the terminal with `subprocess`. I am getting empty results when the process gets called from within the plugin (but works directly from the python shell) 🤔

Reply:
I'll add a config on OM YAML which we will show if present, otherwise we'll run these commands from the airflow host itself